src/main/java/org/dependencytrack/persistence/jdbi/FindingDao.java

```diff
@@ -0,0 +1,435 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) OWASP Foundation. All Rights Reserved.
+ */
+package org.dependencytrack.persistence.jdbi;
+
+import alpine.persistence.PaginatedResult;
+import alpine.resources.AlpineRequest;
+import alpine.server.util.DbUtil;
+import com.github.packageurl.MalformedPackageURLException;
+import com.github.packageurl.PackageURL;
+import org.dependencytrack.model.Finding;
+import org.dependencytrack.model.GroupedFinding;
+import org.dependencytrack.model.RepositoryMetaComponent;
+import org.dependencytrack.model.RepositoryType;
+import org.dependencytrack.persistence.jdbi.mapping.FindingRowMapper;
+import org.dependencytrack.persistence.jdbi.mapping.GroupedFindingRowMapper;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.customizer.AllowUnusedBindings;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.Define;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.dependencytrack.persistence.jdbi.JdbiFactory.withJdbiHandle;
+
+public interface FindingDao {
+
+    @SqlQuery("""
+            SELECT "PROJECT"."UUID" AS "projectUuid"
+                 , "PROJECT"."NAME" AS "projectName"
+                 , "PROJECT"."VERSION" AS "projectVersion"
```

Comment:
Do we need these two? They were not part of the result set before.

Reply:
Project name and version were being mapped [here](https://github.com/DependencyTrack/hyades-apiserver/blob/e7377b179fc525cfb137b6cdb87bbd5c79ad8ab3/src/main/java/org/dependencytrack/model/Finding.java#L220).